src/components/navigation-bar/navigation-bar.test.js

```diff
@@ -0,0 +1,180 @@
+        CypressMountWithProviders(
+          <NavigationBarComponent>{childrenValue}</NavigationBarComponent>
+        );
+
+        navigationBar().should("have.text", childrenValue);
+      }
+    );
+
+    it("should render NavigationBar component with ariaLabel prop", () => {
+      CypressMountWithProviders(
+        <NavigationBarComponent ariaLabel={testData} />
+      );
+
+      navigationBar().should("have.attr", "aria-label", testData);
+    });
+
+    it.each(variants)(
+      "should render NavigationBar component with %s as a navigationType",
+      (navigationType, color) => {
+        CypressMountWithProviders(
+          <NavigationBarComponent navigationType={navigationType} />
+        );
+
+        navigationBar().should("have.css", "background-color", color);
+      }
+    );
+
+    it.each([
+      [true, "not.exist"],
+      [false, "be.visible"],
+    ])(
+      "should render NavigationBar component with isLoading prop set to %s",
+      (boolean, assertion) => {
+        CypressMountWithProviders(
+          <NavigationBarComponent isLoading={boolean}>
+            {testData}
+          </NavigationBarComponent>
+        );
+
+        cy.contains(testData).should(assertion);
+      }
+    );
+
+    it.each(["fixed", "sticky"])(
+      "should render NavigationBar component with position prop set to %s with orientation set to top",
+      (position) => {
+        CypressMountWithProviders(
+          <NavigationBarComponent position={position} orientation="top" />
+        );
+
+        navigationBar().should("have.css", "position", position);
+      }
+    );
+
+    it.each(["fixed", "sticky"])(
+      "should render NavigationBar component with position prop set to %s with orientation set to bottom",
+      (position) => {
+        CypressMountWithProviders(
+          <NavigationBarComponent position={position} orientation="bottom" />
+        );
+
+        navigationBar().should("have.css", "position", position);
+      }
+    );
+
+    describe.each(["top", "bottom"])("orientation %s", (orientation) => {
+      it.each(offsetVal)(
+        "should render NavigationBar component with offset prop set to %s px",
+        (offset) => {
+          CypressMountWithProviders(
+            <NavigationBarComponent
+              offset={`${offset}px`}
+              orientation={orientation}
+              position="fixed"
+            />
+          );
+          navigationBar().then(($el) => {
+            expect($el).to.have.attr("offset").to.equals(`${offset}px`);
+            expect($el).to.have.css(orientation).to.equals(`${offset}px`);
+          });
+        }
+      );
+    });
+
+    it.each(["top", "bottom"])(
+      "should render NavigationBar component with orientation prop set to %s",
+      (orientation) => {
+        CypressMountWithProviders(
+          <NavigationBarComponent orientation={orientation} />
+        );
+
+        navigationBar().should("have.attr", "orientation", orientation);
```

Comment:
```suggestion
          <NavigationBarComponent position="fixed" orientation={orientation} />
        );

        navigationBar().should("have.attr", "position", "fixed");
        navigationBar().should("have.css", orientation, "0px");
```
I would amend this test to do the above since we want to check if the navigation bar is rendered at the top or the bottom of the viewport when a particular value is passed to `orientation`.

Like @robinzigmond mentions in his comment the prop itself is a custom React prop, so there must be a separate bug occurring `orientation` is being rendered on the `<nav>` DOM element.